.github/workflows/universalnrp-test.yml

```diff
@@ -184,6 +184,13 @@ jobs:
               Rename-Item -Path $_.Name -NewName "$name.zip"
           }
       
+      - name: Check package size
+        if: success() || failure()
+        uses: ./.github/actions/check-size
+        with:
+          package: '${{ steps.normalize.outputs.path }}'
+          limit: "290"
```

Comment:
Can `290` be a number?

Reply:
Assuming so. It works fine like this and takes a long time to validate and fix if any regression, so for now I am tempted to leave it like this (make it more obvious where the number starts and ends)

Reply:
Done!